packages/http-server-csharp/src/cli/cli.ts

```diff
@@ -27,10 +29,27 @@ async function main() {
             type: "boolean",
             default: false,
           })
-          .positional("project-directory", {
+          .option("project-name", {
+            description: "The name of the generated project.",
+            type: "string",
+            default: "ServiceProject",
+          })
+          .option("http-port", {
+            description: "The http port for the generated project to use locally",
+            type: "number",
+          })
+          .option("https-port", {
+            description: "The https port for the generated service to listen on locally.",
+            type: "number",
+          })
+          .option("overwrite", {
+            description: "Overwrite existing mock implementations",
```

Comment:
Only mock? Does it touch project files?

Reply:
Yes. all the files marked as 'conditional' are controlled by this.